### ISSUE TITLE
Warning: Calling depends_on :x11 is deprecated! Use depends_on specific X11 formula(e) instead

### ISSUE THREAD

**Issue by luispuerto** [sethrfore/homebrew-r-srf#37]
When `brew update` I get
```bash 
Warning: Calling depends_on :x11 is deprecated! Use depends_on specific X11 formula(e) instead.
Please report this issue to the sethrfore/r-srf tap (not Homebrew/brew or Homebrew/core), or even better, submit a PR to fix it:
  /usr/local/Homebrew/Library/Taps/sethrfore/homebrew-r-srf/r.rb:19
```

I've been taking a look to the formula and so… [it seems][1] that the correct way to fix it is replacing `:X11` by `libx11`. Is this correct? Or do we have a problem here because R still needs XQuarzt to be build? 

[1]: https://github.com/d12frosted/homebrew-emacs-plus/pull/287/commits/135e1f5e81b1e947ecc031d9c851881baaeb0d16

**Comment by luispuerto** [comment]
I tried the `libx11` removing also the xquartz and it didn't work

```bash
==> ./configure --prefix=/usr/local/Cellar/r/4.0.3 --enable-memory-profiling --with-x --with-aqua --
Last 15 lines from /Users/lpuerto/Library/Logs/Homebrew/r/01.configure:
checking whether iswctype exists and is declared... yes
checking for wctrans_t... yes
checking for mbstate_t... yes
checking for ICU... no
checking for ucol_open in -licucore... yes
checking for X... libraries , headers
checking for gethostbyname... yes
checking for connect... yes
checking for remove... yes
checking for shmat... yes
checking for IceConnectionNumber in -lICE... no
checking X11/Intrinsic.h usability... no
checking X11/Intrinsic.h presence... no
checking for X11/Intrinsic.h... no
configure: error: --with-x=yes (default) and X11 headers/libs are not available
```

**Comment by luispuerto** [comment]
Fixed with 591c6ec78680ea5cf4585265467510ea1f746dd3 

Now that there is not `X11` involved,  does this mean we don't need to have installed XQuartz?
